pkgs/development/python-modules/co2signal/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "co2signal";
+  version = "0.4.2";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "CO2Signal";
+    hash = "sha256-8YdYbknLICRrZloGUZuscv5e1LIDZBcCPKZs6EMaNuo=";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  pythonImportsCheck = [ "CO2Signal" ];
+
+  meta = with lib; {
+    description = "A package to access the co2signal API ";
+    homepage = "https://github.com/danielsjf/CO2Signal";
+    license = licenses.gpl3;
```

The LICENSE file says `GPLv3`.

```suggestion
    license = licenses.gpl3Only;
```

The `setup.py` file says [`Creative Commons Attribution-Noncommercial-Share Alike license`](https://github.com/danielsjf/CO2Signal/blob/master/setup.py#L9). Upstream could clarify the actual licensing but I would say that GPLv3 seems right.

---

Indeed, the `setup.py` file from the repo has version `0.4.1` and is clearly licensed under the GPLv3, while the PyPi source has version `0.4.2` and a CC license.